features/commands/exit_statuses.feature

```diff
@@ -0,0 +1,136 @@
+Feature: Check exit status of commands
```

Comment:
Fine. Will change the other "examples" one by one in one of the following PRs. :smile: